### DISCUSSION TITLE
Prometheus metrics ownerApp "Unkown"

### DISCUSSION THREAD

**Discussion by sheinat-ab** [Netflix/conductor#3453] [Q&A]
How can I identify metrics that are being sent to a prometheus server, many of which have appOwner "Unknown"?

Thanks!

**Comment by DukeDai** [comment]
first of all, actuator/prometheus is enabled, then wether there is scrapper to grab and send to prometheus. 

Try query with your env and app name, or simply remove tag criteria: 
`sum by(workflowName) (rate(workflow_running{env="$env", app="conductor-server"}[$__rate_interval]))`

`sum by(workflowName) (rate(workflow_running[$__rate_interval]))`
